pkg/drivers/kic/oci/oci.go

```diff
@@ -185,12 +185,40 @@ func CreateContainerNode(p CreateParams) error { //nolint to suppress cyclomatic
 		// label th enode with the node ID
 		"--label", p.NodeLabel,
 	}
-	// to provide a static IP
-	if p.Network != "" && p.IP != "" {
+
+	// attach to the user-defined bridge network (once), and set static IPs if provided
+	if p.Network != "" {
 		runArgs = append(runArgs, "--network", p.Network)
-		runArgs = append(runArgs, "--ip", p.IP)
+		if p.IP != "" {
+			runArgs = append(runArgs, "--ip", p.IP) // IPv4
+		}
+		if p.IPv6 != "" {
+			runArgs = append(runArgs, "--ip6", p.IPv6) // IPv6
+		}
+	}
+
+	// For IPv6/dual clusters, enable forwarding inside the node container
+	// (safe sysctl; avoid disable_ipv6 which may be blocked by Docker's safe list)
+
+	// Ensure service rules apply to bridged traffic inside the node container.
+	// Do both families; harmless if already set.
+	runArgs = append(runArgs,
+		"--sysctl", "net.ipv4.ip_forward=1",
+		"--sysctl", "net.bridge.bridge-nf-call-iptables=1",
```

Comment:
Can you please document/explain a bit more why "net.bridge.bridge-nf-call-iptables=1" is set?

This requires the br_netfilter kernel module? Do we need to check if it's loaded, or can we assume it's loaded?